include/tools.h

```diff
@@ -282,5 +282,14 @@ FeedCategories readCategoriesFromFeed(const std::string& content);
  * @return full language name.
  */
 std::string getLanguageSelfName(const std::string& lang);
+
+/**
+ * Slugifies the filename by converting any characters reserved by the operating
+ * system to '_'.
+ *
+ * @param filename file name string.
+ * @return slugified string.
```

Comment:
I think it should be emphasized that the input is a file name, not a filepath

Reply:
Please also mention that this should be a valid utf8 encoded string.

Windows use different encoding. It is better to be explicit here that we want utf8.

Reply:
Done